packages/terra-list/src/ListSectionHeader.jsx

```diff
@@ -107,6 +121,12 @@ const ListSectionHeader = ({
     attrSpread.onBlur = ListUtils.wrappedEventCallback(onBlur, event => event.currentTarget.setAttribute('data-item-show-focus', 'true'));
     attrSpread.onMouseDown = ListUtils.wrappedEventCallback(onMouseDown, event => event.currentTarget.setAttribute('data-item-show-focus', 'false'));
   }
+  if (role && role.length > 0 && role !== 'none') {
```

Comment:
This can be cleaned up if we don't default a role. A simple
```
if (role) {
```
would suffice.

Reply:
Changed cf4a187